File: typescript/src/file.ts
import path from 'path';
import { fileURLToPath } from 'url';

const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);

export function getDataFilePath(fileName: string) {
  return path.join(__dirname, `../../data/${fileName}`);
}

File: python/src/file.py
import os


def get_data_file_path(file_name: str) -> str:
    file_path = os.path.join(os.path.dirname(__file__), "../../data", file_name)
    if not os.path.exists(file_path):
        print(f"- File {file_path} does not exist")
        exit(1)
    return file_path
